### ISSUE TITLE
DebugBitstream does not work when RateControl enabled

### ISSUE THREAD

**Issue by siwamura** [fraunhoferhhi/vvenc#119]
When running vvencFFapp with DebugBitstream option, encoder crashes after decoding POC0.

`vvencFFapp -c randomaccess_medium.cfg -c sequence.cfg --DebugBitstream=old_str.bin -q 37 -c rc1p.cfg` ,
where `old_str.bin` was created by:
`vvencFFapp -c randomaccess_medium.cfg -c sequence.cfg -b old_str.bin -q 37 -c rc1p.cfg`.
 
This may be because initialization of  encRcPic done in EncPicture::compressPicture() is skipped when trySkipOrDecodePicture() successfully decodes current POC.

**Comment by adamjw24** [comment]
Did it ever worked with rate control (on previous versions for example)?

I think this a corner case we never tested.

We'll have a look, but it would be great to know if its a new issue or if it was just only now found.

**Comment by siwamura** [comment]
This crash was also identified in the previous versions such as v1.1.0 and v0.3.1.0.
It would be really appreciated if this issue could be resolved.

**Comment by adamjw24** [comment]
We had a look into this, and it would be a major task to get it working.

In 1.3 we included a new single pass RC mode, which will become default in the next major version (for now can be used with `--LookAhead=1`), which also renders fixing it for current single pass obsolete.

For 1.3.1, we disabled this use-case with a parameter check, since there are also still issues with the new rate control.

We'll try to get it working with the new single-pass RC mode as well as two-pass for the next version.

Sorry there's no better news.
